pallets/common/src/lib.rs

```diff
@@ -1222,7 +1222,8 @@ impl<T: Config> Pallet<T> {
 		Ok(())
 	}
 
-	/// Set scouped collection property.
+	/// Set a scoped collection property, where the scope is a special prefix
+	/// prohibiting a user to change the property.
```

Comment:
И аналогично в других местах.

Хотелось бы подчеркнуть, что не вообще запрещает, а только напрямую.

```suggestion
	/// prohibiting a user from changing the property directly.
```